src/components/ExportFromTextButton.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {downloadFromText} from "../utils/download";
+
+import {Button} from "antd";
+import "antd/es/button/style/css";
+
+import { DownloadOutlined } from "@ant-design/icons";
+
+
+export const ExportFromTextButton = ({ exportFunction, filename }) => {
+  let name = filename + '_' + new Date().toISOString().slice(0, 10) + '.csv'
+  const onClick = () => {
+    exportFunction()
+      .then(text => {
+        downloadFromText(name, text)
+      })
+  }
+  return (
+    <Button onClick={onClick}>
+      <DownloadOutlined />
+      Export
+    </Button>
+  )
+}
```

Comment:
Components should be exported as the default export, not as a named export.

Reply:
Also now that it's the only type of ExportButton there is no need anymore to include "FromText" in its name.